system-tests/services/test-external-volumes.js

```diff
@@ -0,0 +1,257 @@
+require("../_support/utils/ServicesUtil");
+const { Timeouts } = require("../_support/constants");
+
+describe("Services", function() {
+  /**
+   * Test the external volumes
+   *
+   * --- IMPORTANT NOTICE ---
+   *
+   * REMEMBER TO UPDATE num_vols_delete IN external-volumes/teardown
+   * WHENEVER YOU ADD OR REMOVE A TEST CASE
+   */
+  describe("External Volumes", function() {
+    afterEach(() => {
+      cy.window().then(win => {
+        win.location.href = "about:blank";
+      });
+    });
+
+    it("should create an app with external volume", function() {
```

Comment:
we are removing `should` from tests when writing new ones, this is not a big deal but would be nice not to use it anymore.